TensorFlow2/Segmentation/MaskRCNN/mask_rcnn_sm.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Scripts that simplifies running training benchmark """
+
+import argparse
+import os
+import shutil
+import subprocess
+from argparse import ArgumentError
+
+
+def csv_int(vstr, sep=','):
+    values = []
+    for v0 in vstr.split(sep):
+        try:
+            v = int(v0)
+            values.append(v)
+        except ValueError as err:
+            raise ArgumentError('Invalid value %s, values must be a number' % v)
+    return values
+
+def main():
+    # CLI flags
+    print ("Here")
+    parser = argparse.ArgumentParser(description="MaskRCNN train benchmark")
+    parser.add_argument('--mode', type=str, default='train')
```

Comment:
Should we add the description from `mask_rcnn_main.py` file here ?

Reply:
We can add those. I'll do it in a separate PR. 
For now, let's get this to run on SM :)

Reply:
Sure